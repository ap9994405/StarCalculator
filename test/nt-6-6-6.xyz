144
Nanotube (6,6) with 6 unit cells
C       4.06800    0.00000    0.00000
C       3.82267    1.39134    0.00000
C       2.03400    3.52299    0.00000
C       0.70640    4.00620    0.00000
C      -2.03400    3.52299    0.00000
C      -3.11627    2.61486    0.00000
C      -4.06800    0.00000    0.00000
C      -3.82267   -1.39134    0.00000
C      -2.03400   -3.52299    0.00000
C      -0.70640   -4.00620    0.00000
C       2.03400   -3.52299    0.00000
C       3.11627   -2.61486    0.00000
C       3.52299    2.03400    1.22976
C       2.61486    3.11627    1.22976
C       0.00000    4.06800    1.22976
C      -1.39134    3.82267    1.22976
C      -3.52299    2.03400    1.22976
C      -4.00620    0.70640    1.22976
C      -3.52299   -2.03400    1.22976
C      -2.61486   -3.11627    1.22976
C      -0.00000   -4.06800    1.22976
C       1.39134   -3.82267    1.22976
C       3.52299   -2.03400    1.22976
C       4.00620   -0.70640    1.22976
C       2.03400    3.52299    2.45951
C       0.70640    4.00620    2.45951
C      -2.03400    3.52299    2.45951
C      -3.11627    2.61486    2.45951
C      -4.06800    0.00000    2.45951
C      -3.82267   -1.39134    2.45951
C      -2.03400   -3.52299    2.45951
C      -0.70640   -4.00620    2.45951
C       2.03400   -3.52299    2.45951
C       3.11627   -2.61486    2.45951
C       4.06800    0.00000    2.45951
C       3.82267    1.39134    2.45951
C       0.00000    4.06800    3.68927
C      -1.39134    3.82267    3.68927
C      -3.52299    2.03400    3.68927
C      -4.00620    0.70640    3.68927
C      -3.52299   -2.03400    3.68927
C      -2.61486   -3.11627    3.68927
C      -0.00000   -4.06800    3.68927
C       1.39134   -3.82267    3.68927
C       3.52299   -2.03400    3.68927
C       4.00620   -0.70640    3.68927
C       3.52299    2.03400    3.68927
C       2.61486    3.11627    3.68927
C      -2.03400    3.52299    4.91902
C      -3.11627    2.61486    4.91902
C      -4.06800    0.00000    4.91902
C      -3.82267   -1.39134    4.91902
C      -2.03400   -3.52299    4.91902
C      -0.70640   -4.00620    4.91902
C       2.03400   -3.52299    4.91902
C       3.11627   -2.61486    4.91902
C       4.06800    0.00000    4.91902
C       3.82267    1.39134    4.91902
C       2.03400    3.52299    4.91902
C       0.70640    4.00620    4.91902
C      -3.52299    2.03400    6.14878
C      -4.00620    0.70640    6.14878
C      -3.52299   -2.03400    6.14878
C      -2.61486   -3.11627    6.14878
C      -0.00000   -4.06800    6.14878
C       1.39134   -3.82267    6.14878
C       3.52299   -2.03400    6.14878
C       4.00620   -0.70640    6.14878
C       3.52299    2.03400    6.14878
C       2.61486    3.11627    6.14878
C      -0.00000    4.06800    6.14878
C      -1.39134    3.82267    6.14878
C      -4.06800    0.00000    7.37854
C      -3.82267   -1.39134    7.37854
C      -2.03400   -3.52299    7.37854
C      -0.70640   -4.00620    7.37854
C       2.03400   -3.52299    7.37854
C       3.11627   -2.61486    7.37854
C       4.06800    0.00000    7.37854
C       3.82267    1.39134    7.37854
C       2.03400    3.52299    7.37854
C       0.70640    4.00620    7.37854
C      -2.03400    3.52299    7.37854
C      -3.11627    2.61486    7.37854
C      -3.52299   -2.03400    8.60829
C      -2.61486   -3.11627    8.60829
C      -0.00000   -4.06800    8.60829
C       1.39134   -3.82267    8.60829
C       3.52299   -2.03400    8.60829
C       4.00620   -0.70640    8.60829
C       3.52299    2.03400    8.60829
C       2.61486    3.11627    8.60829
C      -0.00000    4.06800    8.60829
C      -1.39134    3.82267    8.60829
C      -3.52299    2.03400    8.60829
C      -4.00620    0.70640    8.60829
C      -2.03400   -3.52299    9.83805
C      -0.70640   -4.00620    9.83805
C       2.03400   -3.52299    9.83805
C       3.11627   -2.61486    9.83805
C       4.06800    0.00000    9.83805
C       3.82267    1.39134    9.83805
C       2.03400    3.52299    9.83805
C       0.70640    4.00620    9.83805
C      -2.03400    3.52299    9.83805
C      -3.11627    2.61486    9.83805
C      -4.06800    0.00000    9.83805
C      -3.82267   -1.39134    9.83805
C      -0.00000   -4.06800   11.06780
C       1.39134   -3.82267   11.06780
C       3.52299   -2.03400   11.06780
C       4.00620   -0.70640   11.06780
C       3.52299    2.03400   11.06780
C       2.61486    3.11627   11.06780
C      -0.00000    4.06800   11.06780
C      -1.39134    3.82267   11.06780
C      -3.52299    2.03400   11.06780
C      -4.00620    0.70640   11.06780
C      -3.52299   -2.03400   11.06780
C      -2.61486   -3.11627   11.06780
C       2.03400   -3.52299   12.29756
C       3.11627   -2.61486   12.29756
C       4.06800    0.00000   12.29756
C       3.82267    1.39134   12.29756
C       2.03400    3.52299   12.29756
C       0.70640    4.00620   12.29756
C      -2.03400    3.52299   12.29756
C      -3.11627    2.61486   12.29756
C      -4.06800    0.00000   12.29756
C      -3.82267   -1.39134   12.29756
C      -2.03400   -3.52299   12.29756
C      -0.70640   -4.00620   12.29756
C       3.52299   -2.03400   13.52732
C       4.00620   -0.70640   13.52732
C       3.52299    2.03400   13.52732
C       2.61486    3.11627   13.52732
C      -0.00000    4.06800   13.52732
C      -1.39134    3.82267   13.52732
C      -3.52299    2.03400   13.52732
C      -4.00620    0.70640   13.52732
C      -3.52299   -2.03400   13.52732
C      -2.61486   -3.11627   13.52732
C      -0.00000   -4.06800   13.52732
C       1.39134   -3.82267   13.52732
