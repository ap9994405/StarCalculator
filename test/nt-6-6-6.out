 1999480625 + 20761906560 x + 101508439704 x^2 + 310998309692 x^3 + 670462857750 x^4 + 1082835732852 x^5 + 1362171480554 x^6 + 1370269359588 x^7 + 1123178314650 x^8 + 760755688044 x^9 + 430422504486 x^10 + 205185305460 x^11 + 82997954724 x^12 + 28653801708 x^13 + 8481233568 x^14 + 2158393420 x^15 + 472439241 x^16 + 88652400 x^17 + 14145570 x^18 + 1891332 x^19 + 206946 x^20 + 17848 x^21 + 1140 x^22 + 48 x^23 + 1 x^24
 total: 7563418117911
